release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -206,6 +206,9 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-7-removed-features"]
 === Removed features
 
+==== Installer-provisioned clusters no longer require provisioningHostIP or bootstrapProvisioningIP
```

Comment:
@johnwilkins This subheading needs its own section ID. Other than that, this looks good!